### PR TITLE
MWPW-134213 [MILO] Refactor buildInternalBanner inside scripts.js so that it doesn't pull the whole query index

### PR DESCRIPTION
**Pull request by sharmrj**
https://jira.corp.adobe.com/browse/MWPW-134213

### CODE REVIEW

File: scripts/scripts.js [hunk 1]
```diff
@@ -286,17 +286,18 @@ async function buildInternalBanner() {
     text.innerHTML = '&nbsp;';
     banner.append(text);
 
+    // get last updated date from the http header
```
Comment (nkthakur48) [inline scripts/scripts.js:289]: Why we are making call to query-index?
Screenshot 2023-07-26 at 10 47 07 AM

**Review by nkthakur48 — CHANGES_REQUESTED** [review]

File: scripts/scripts.js (hunk 1 above)
Comment (sharmrj) [reply scripts/scripts.js:289]: I missed that line. It's fixed now.

**Review by adisharm — DISMISSED** [review]

File: scripts/scripts.js [hunk 2]
```diff
@@ -286,17 +286,17 @@ async function buildInternalBanner() {
...
-      const placeholders = await fetchPlaceholders();
-      const dateFormat = new Date(parseInt(`${found.lastModified}000`, 10));
-      const productNames = document.querySelector('meta[name="productnames"]')?.content.split(',');
-      const primary = document.querySelector('meta[name="primaryproductname"]')?.content;
-      const alsoAppliesTo = productNames?.length ? ` | Also Applies to ${productNames.filter(x => x !== primary).join(', ')} ` : '';
-      text.innerHTML = 
-        `${placeholders.lastUpdatedOn || 'Last updated on'} ${getMonthShortName((dateFormat.getMonth()))} ${dateFormat.getDate()}, ${dateFormat.getFullYear()}${alsoAppliesTo}`;
-    }
+    // get last updated date from the http header
+    const req = new XMLHttpRequest();
```
Comment (nkthakur48) [inline scripts/scripts.js]: Could we do `fetch` syntax here
```javascript
fetch(url, {method: 'HEAD'})
```

**Review by nkthakur48 — CHANGES_REQUESTED** [review]

File: scripts/scripts.js (hunk 2 above)
Comment (sharmrj) [reply scripts/scripts.js]: Done

**Review by nkthakur48 — APPROVED** [review]

**Pull request merged**